code/datums/religions.dm

```diff
@@ -467,7 +469,7 @@
 	bible_names = list("The Vedas", "The Mahabharata")
 	male_adept = "Guru"
 	female_adept = "Guru"
-	keys = list("hindu", "hinduism", "india")
+	keys = list("hindu", "hinduism", "india", "designated")
```

Comment:
kek

Reply:
oh i was gonna go all in but i dont know if either github or people here are fully ok with it

Reply:
your good bro

Reply:
dun